### DISCUSSION TITLE
Can I install without Android Studio if I don't target Android?

### DISCUSSION THREAD

**Discussion by geronimus** [dotnet/maui#6699] [Q&A] [answered]
I realize I'm in the minority, but I'm not interested in deploying to Android. I'd just like to use my personal computer - which is a Mac - to develop desktop apps to use at work - where Windows and Linux are the norm.

Is it possible, then, to make .NET MAUI on Mac work without the Android Studio prerequisite?

(Flutter, for example, doesn't work properly without Android Studio, no matter which OS is the target.)

**Comment by kcrg** [comment, marked as answer]
Android Studio is not required, you can use MAUI without AS just fine, even to build and deploy Android app you don't need Android Studio.
